### DISCUSSION TITLE
创建demo workspace 默认启用云同步，应该怎么做？谢谢

### DISCUSSION THREAD

**Discussion by chenmins** [toeverything/AFFiNE#1757] [Q&A]
​preset.config.mjs enableIndexedDBProvider 的作用是什么？
如果 我希望用户登录后，默认创建的工作空间自动云同步，
那么enableIndexedDBProvider 设置为false是否可以不启用indexdb的本地功能？
是否可以在创建demo workspace是启用云同步
如果不能，请问在哪里修改
谢谢

**Comment by HeJiachen-PM** [comment]
Hi, we closed AFFiNE cloud sign-up temporarily because it is under major refactoring. The existing AFFiNE cloud user may still access the AFFiNE cloud in app.affine.pro. We are trying our best to offer a good AFFiNE cloud in next major release (0.7.0)
